.gitignore

```diff
@@ -3,3 +3,4 @@ htmlcov/
 coverage/
 .coverage
 *.kdev4
+build/
```

Comment:
I have `test_python/build*` in `documentation/.gitignore`, and that was sufficient for me. Does it not work for you?